mocktracer/include/opentracing/mocktracer/json.h

```diff
@@ -0,0 +1,20 @@
+#ifndef OPENTRACING_MOCKTRACER_JSON_H
+#define OPENTRACING_MOCKTRACER_JSON_H
+
+#include <opentracing/mocktracer/recorder.h>
+#include <vector>
+
+namespace opentracing {
+BEGIN_OPENTRACING_ABI_NAMESPACE
+namespace mocktracer {
+// Serialize provided spans to JSON.
+std::string ToJson(const std::vector<SpanData>& spans);
```

Comment:
If you want better nlohmann/JSON support, put this in the `nlohmann` namespace as described in the docs there. That allows for native conversion to/from JSON types.

Reply:
Never mind see you did it later.